src/QsCompiler/Transformations/CallGraphWalker.cs

```diff
@@ -13,515 +13,10 @@
 
 // ToDo: Review access modifiers
 
-namespace Microsoft.Quantum.QsCompiler.Transformations
+namespace Microsoft.Quantum.QsCompiler.Transformations.CallGraphNS
```

Comment:
The name ending in *NS seems a bit odd to me (any namespace could/should then end in NS). Is it intended to stay?

Reply:
I got the idea that ending a namespace with NS was a MS standard with single class namespaces from this post: https://stackoverflow.com/a/48113597/5399520
After looking into it a bit more, I'm not sure that is the case. I dropped the NS from the namespace and rewrote the class.

Reply:
These are some unusual transformations because they are static extension methods. I noticed this transformation class did not have a namespace for itself under `Transformations`, so I added one to fix that. But with them being extension methods, I am not sure. Is this where all extension methods that will be transformation will go?

Reply:
I decided to drop the CodeTransformations namespace because that's what made sense to me. I kept the rename for the CodeTransformations class to CodeTransformationExtensions because that is the convention for static classes that just provide extension methods, and it makes it more clear that that is what that class does.